src/scrapers/hapoalim.js

```diff
@@ -84,6 +88,19 @@ async function fetchPoalimXSRFWithinPage(page, url, pageUuid) {
   return fetchPostWithinPage(page, url, [], headers);
 }
 
+export async function fetchPoalimXSRFWithinPageGet(page, url, pageUuid) {
```

Comment:
probably can share code with `fetchPoalimXSRFWithinPage` but because I don't have anyone to test regular Poalim with me, I preferred to do it like that for the beginning

Reply:
it might need it when we'll get all the possible information from the bank's website like I do in the business one